util/chunk/codec.go

```diff
@@ -140,6 +140,7 @@ func (c *Codec) decodeColumn(buffer []byte, col *Column, ordinal int) (remained
 
 	// decode data.
 	col.data = buffer[:numDataBytes:numDataBytes]
+	// the column reference the data of grpc, so need set col.avoidReusing to avoid reusing it.
```

Comment:
The column reference the data of the grpc response, the memory of the grpc message cannot be GCed if we reuse this column. Thus we set \`avoidResuing\` to true.